portal-api/src/v1/services/utilisation-report/email-service.ts

```diff
@@ -17,17 +17,34 @@ const getUkefGefReportingEmailRecipients = () => {
   }
 };
 
+/**
+ * Retrieves the TFM UI URL from the environment variables.
+ * @returns The TFM UI URL.
+ * @throws {InvalidEnvironmentVariableError} If the TFM_UI_URL environment variable is not defined or is empty.
+ */
+export const getTfmUiUrl = (): string => {
+  const tfmUiUrl = process.env.TFM_UI_URL;
```

Comment:
```suggestion
  const { TFM_UI_URL } = process.env;
```

Reply:
Updated thank you

Reply:
_PR approved following this change, this was the only comment, resolving so I can merge_